runtime/include/Types.h

```diff
@@ -26,12 +26,12 @@ extern "C" {
 
 // Qubit, Result and Observable types
 struct QUBIT;
-typedef intptr_t QubitIdType;
+using QubitIdType = intptr_t;
```

Comment:
Probably shouldn't be using C++ in our C code 😅

Reply:
Ah! Forgot to revert these suggestions... we're not going to linting C code using clang-tidy C++ suggestions :D